test/ci-cd.spec.js

```diff
@@ -17,264 +17,572 @@ describe('JHipster CI-CD Sub Generator', () => {
     //--------------------------------------------------
     // Jenkins tests
     //--------------------------------------------------
-
-    describe('Jenkins: maven AngularX NPM', () => {
-        before(done => {
-            helpers
-                .run(require.resolve('../generators/ci-cd'))
-                .inTmpDir(dir => {
-                    fse.copySync(path.join(__dirname, './templates/ci-cd/maven-ngx-npm'), dir);
-                })
-                .withOptions({ skipChecks: true })
-                .withPrompts({
-                    pipeline: 'jenkins',
-                    insideDocker: false,
-                    cicdIntegrations: [],
-                })
-                .on('end', done);
-        });
-        it('creates expected files', () => {
-            assert.file(expectedFiles.jenkins);
+    describe('Jenkins tests', () => {
+        describe('Jenkins: Maven AngularX NPM', () => {
```

Comment:
Can we get rid of AngularX? I think we use the X in a lot of places.

Reply:
just done :)